Examples/Docs/Sources/SyncExamples/main.swift

```diff
@@ -12,8 +12,8 @@ private func causalConsistency() throws {
     let s1 = client1.startSession(options: ClientSessionOptions(causalConsistency: true))
     let currentDate = Date()
     var dbOptions = DatabaseOptions(
-        readConcern: ReadConcern(.majority),
-        writeConcern: try WriteConcern(w: .majority, wtimeoutMS: 1000)
+        readConcern: .majority,
+        writeConcern: try WriteConcern.majority(wtimeoutMS: 1000)
```

Comment:
I think this can still be `.majority(wtimeoutMS: 1000)`, though I could be wrong.

Reply:
Indeed it can! I wish the linter could pick up on this somehow, maybe someday